pkg/nvidiagpu/consts.go

```
@@ -1,11 +1,24 @@
+)
 
 const (
 	NvidiaGPUNamespace = "nvidia-gpu-operator"
 
-	NvidiaGPULabel                   = "feature.node.kubernetes.io/pci-10de.present"
+	// NvidiaGPULabel is the legacy NFD label for NVIDIA GPU presence (vendor ID only, pre-4.21).
+	NvidiaGPULabel = "feature.node.kubernetes.io/pci-10de.present"
+	// nfdLabelPrefix is the common prefix for NFD PCI labels.
+	nfdLabelPrefix = "feature.node.kubernetes.io/pci-"
+	// nfdNvidiaVendorSuffix is the NVIDIA vendor ID suffix in NFD PCI labels.
+	nfdNvidiaVendorSuffix = "_10de.present"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)